Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.18.1 as builder
```

Comment:
Follow Up: we should have a base dockerfile that sets the golang version so that we ensure all golang repos are using the same golang version.

Reply:
I don't think that's a good idea, we should let every image decide the version (but keep it updated with dependabot) otherwise we would be vulnerable to breaking something when upgrading the version across all repos without properly testing them.